### ISSUE TITLE
Is there a special export format?

### ISSUE THREAD

**Issue by guhur** [akmtn/pytorch-siggraph2017-inpainting#1]
I tried the method on another image, but it has completely failed. I know you did not write the paper, but maybe you have some insights about it:

![image](https://user-images.githubusercontent.com/12297742/48561009-5a067300-e8ef-11e8-820b-6e3b56861eef.png)
Before

![image](https://user-images.githubusercontent.com/12297742/48561016-5ffc5400-e8ef-11e8-8bd2-661fff4ceac3.png)
Mask

![image](https://user-images.githubusercontent.com/12297742/48560997-51ae3800-e8ef-11e8-9c21-e3cafc5131e3.png)
After

**Comment by guhur** [comment]
This seems to be a normal behavior of the neural network. One should avoid large masks and use outdoor images.
